src/sections/Home/Partners-home/index.js

```diff
@@ -42,15 +72,17 @@ const Projects = () => {
           </SectionTitle>
         </Row>
       </Container>
-      <Slider {...settings}>
-        {partners.map((partner, index) => (
-          <Link className="partner-card" to={partner.imageRoute} key={index}>
-            <div className={partner.innerDivStyle}>
-              <img className="partner-image" id={partner.name} loading="lazy" src={partner.imageLink} alt={partner.name} width={partner.imageWidth} height={partner.imageHeight} />
-            </div>
-          </Link>
-        ))}
-      </Slider>
+      {imagesLoaded ? (
+        <Slider {...settings}>
+          {partners.map((partner, index) => (
+            <Link className="partner-card" to={partner.imageRoute} key={index}>
+              <PartnerImage partner={partner} />
+            </Link>
+          ))}
+        </Slider>
+      ) : (
+        <div style={{ textAlign: "center", padding: "20px" }}>Loading partners...</div>
```

Comment:
we don't need to display a fallback as a message in our homepage this is not good. Do not add a fallback here. keep it as it was earlier and just fix smoothness of transitions